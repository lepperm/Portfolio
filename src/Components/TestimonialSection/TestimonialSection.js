/**
 * @fileoverview TestimonialSection component.
 */

import React from "react";
import PropTypes from "prop-types";

/**
 * TestimonialSection component
 *
 * @return {Component} TestimonialSection component
 */
export const TestimonialSection = () => {
  return (
    <section>
      Does it even make sense to have a testimonials section for myself? ðŸ¤”
    </section>
  );
};

/**
 * Prop types for TestimonialSection component
 */
TestimonialSection.propTypes = {
  //
};
