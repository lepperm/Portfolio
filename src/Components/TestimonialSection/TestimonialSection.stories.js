/**
 * @fileoverview Story for TestimonialSection component.
 */

import React from "react";
import { TestimonialSection } from "./TestimonialSection";

export default {
  title: "TestimonialSection",
  component: TestimonialSection,
};

export const Basic = () => <TestimonialSection />;
