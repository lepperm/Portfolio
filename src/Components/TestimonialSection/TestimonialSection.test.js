/**
 * @fileoverview Tests for TestimonialSection component.
 */

import React from "react";
import { shallow } from "enzyme";
import { TestimonialSection } from "./TestimonialSection";

describe("TestimonialSection", () => {
  test("matches snapshot", () => {
    const wrapper = shallow(<TestimonialSection />);
    expect(wrapper).toMatchSnapshot();
  });
});
