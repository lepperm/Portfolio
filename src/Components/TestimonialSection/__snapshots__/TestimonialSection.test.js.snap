// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestimonialSection matches snapshot 1`] = `ShallowWrapper {}`;
